devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -145,6 +145,72 @@ protected SpanByte SliceGenericBuffer(int start, int length)
             return _genericBuffer;
         }
 
+        /// <summary>
+        /// Copies buffer content directly to display buffer.
+        /// Y and height must be byte aligned because buffer will 
+        /// be copied without any logical operations on existing content.
+        /// </summary>
+        /// <param name="x">The x coordinate on the screen.</param>
+        /// <param name="y">The y coordinate on the screen.</param>
+        /// <param name="width">Width of buffer content in pixels.</param>
+        /// <param name="height">Height of buffer content in pixels.</param>
+        /// <param name="buffer">Data to copy.</param>
```

Comment:
please with the size the buffer needs to have to make it clear. In short, height * width / 8.